woudc_data_registry/models.py

```diff
@@ -278,13 +278,17 @@ class Instrument(base):
     __tablename__ = 'instruments'
 
     id_field = 'instrument_id'
-    id_dependencies = ['name', 'model', 'serial', 'station_id', 'dataset_id']
+    id_dependencies = ['name', 'model', 'serial',
+                       'station_id', 'dataset_id', 'contributor']
 
     instrument_id = Column(String, primary_key=True)
     station_id = Column(String, ForeignKey('stations.station_id'),
```

Comment:
Since station_id is accessible by joining with deployment, this station_id is redundant.
However, redundancy is sometimes good because it makes queries shorter and more convenient. So you can keep this if you want, just letting you know that removing it is also an option.

Maybe Tom has an opinion. If you do delete it, remember to update `epicenter/instrument` too.